views/AppView.js

```diff
@@ -0,0 +1,34 @@
+var AppView = Backbone.View.extend({
+  el: '.video-search',
+
+  events: {
+    'click #submit': 'searchVideo'
+  },
+
+  initialize: function() {
+    this.$input = this.$('.search');
+    this.$mainVideo = this.$('#mainVideo');
+    this.listenTo(VideoCollection, 'change', this.renderVideo);
```

Comment:
this.model.videos rather than VideoCollection.  VideoCollection is the definition, not the instance.